tests/styleguide.spec.ts

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable no-await-in-loop */
+import { expect, Locator, Page, test } from '@playwright/test';
+import { logBrowserInfo } from './utils';
+
+let page: Page;
+let sampleSections: Locator;
+
+test.beforeEach(async ({ browser }) => {
+  logBrowserInfo(browser);
+
+  page = await browser.newPage();
+  await page.goto('/ide/styleguide');
+
+  sampleSections = page.locator('.sample-section');
+  await expect(sampleSections).toHaveCount(39);
+});
+
+test.afterEach(async () => {
+  await page.close();
+});
+
+// Iterate over all sample sections and take a screenshot of each one.
+test('UI regression test', async () => {
+  for (let i = 0; i < (await sampleSections.count()); i += 1) {
```

Comment:
Should only need to get the count once:
```suggestion
  const sectionCount = await sampleSections.count();
  for (let i = 0; i < sectionCount; i += 1) {
```

Reply:
Fixed